ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -440,6 +468,98 @@ export default {
         }
       },
     },
+    {
+      id: 'E12S Promise Chiseled Sculpture Tether',
+      netRegex: NetRegexes.tether({ target: 'Chiseled Sculpture', id: '0011' }),
+      promise: async (data, matches) => {
```

Comment:
Hmm, would you mind switching this to not use `getCombatants`?  You can get the positions of the statues from the `Classical Sculpture` ability: https://github.com/quisquous/cactbot/blob/main/ui/oopsyraidsy/data/05-shb/raid/e12s.js#L89-L100

I think it's more reliable long-term to do this from just log lines directly.

Reply:
The `Classical Sculpture` ability always shows source entity position as center of arena (0, 75), I'm guessing because damage is triggered before the entities are moved to healer positions.

```
22|2021-01-23T23:28:38.1450000-05:00|400078F5|Chiseled Sculpture|58B2|Classical Sculpture|********|***** *****|F1730006|1F0D4001|F40E|82A0000|1B|58B28000|0|0|0|0|0|0|0|0|0|0|127297|127297|3839|10000|0|1000|0.1265127|-72.03751|75|-0.04555082|148000|148000|0|10000|0|1000|0|-75|75|-3.141593|00004D63|9e34592d7df884fbb12951a55482ff5e
```

I guess it could go based on healer position, maybe? (e.g. trigger on 58B2, early return if target isn't a healer, otherwise store it in statues array as before)

Reply:
I have seen this in my logs:
```
[21:27:42.609] 16:4000DA5F:Chiseled Sculpture:58B2:Classical Sculpture:id:name:730003:6D214001:F40E:82A0000:1B:58B28000:0:0:0:0:0:0:0:0:0:0:140477:148513:10000:10000:0:1000:-0.2594604:-59.06775:75:3.117912:148000:148000:0:10000:0:1000:-0.6748441:-58.87721:75:3.099695:00004663
```

Reply:
Might be latency-dependant or something. Looking at about 15 different pulls, 7 of them had center coordinates for me, including two pulls where one statue had center coords and the other had correct coords. These are from the log files themselves.

Reply:
Here's a gist of the raw numbers I was looking at:
https://gist.github.com/valarnin/5be37cbf53c153236005206fcd7fa1a6

Reply:
Ugh, that's unfortunate.  Thanks for the data.  This is another for the pile of "positional data in abilities is flaky" causing headaches.  I guess I need to fix my oopsy triggers as well to use getCombatants, but there's not support for promises there.

Reply:
As I said before, this could probably be addressed by filtering to just the two healer positions, but given that position data is inconsistent I'm leery about trusting that as well.